### PR TITLE
Add Material Design color palette

### PR DESCRIPTION
**Pull request by drinfernoo**

### CODE REVIEW

File: 1080i/script-skin_helper_service-ColorPicker.xml [hunk 1]
```diff
@@ -30,28 +30,28 @@
                 <scrolltime>200</scrolltime>
                 <orientation>vertical</orientation>
                 <preloaditems>2</preloaditems>
-                <itemlayout width="150" height="150">
+                <itemlayout width="128" height="128">
```
Comment (jurialmunkey) [inline 1080i/script-skin_helper_service-ColorPicker.xml]: Can you change this to `<itemlayout width="100" height="100">`

File: 1080i/script-skin_helper_service-ColorPicker.xml [hunk 2]
```diff
@@ -30,28 +30,28 @@
...
                     <control type="image">
                         <left>0</left>
                         <top>0</top>
-                        <right>10</right>
-                        <bottom>10</bottom>
+                        <right>8</right>
+                        <bottom>8</bottom>
```
Comment (jurialmunkey) [inline 1080i/script-skin_helper_service-ColorPicker.xml]: Can you revert the right/bottom padding back to 10

File: 1080i/script-skin_helper_service-ColorPicker.xml [hunk 3]
```diff
@@ -30,28 +30,28 @@
...
                     </control>
                     <control type="image">
-                        <top>10</top>
-                        <left>10</left>
-                        <right>20</right>
-                        <bottom>20</bottom>
+                        <top>8</top>
+                        <left>8</left>
+                        <right>16</right>
+                        <bottom>16</bottom>
```
Comment (jurialmunkey) [inline 1080i/script-skin_helper_service-ColorPicker.xml]: Same here. Can you revert the padding top/left pad to 10 and right/bottom pad to 20

File: 1080i/script-skin_helper_service-ColorPicker.xml [hunk 4]
```diff
@@ -30,28 +30,28 @@
...
                         <texture background="true">$INFO[ListItem.Icon]</texture>
                     </control>
                 </itemlayout>
-                <focusedlayout width="150" height="150">
+                <focusedlayout width="128" height="128">
```
Comment (jurialmunkey) [inline 1080i/script-skin_helper_service-ColorPicker.xml]: Same here. Can you change to `<focusedlayout width="100" height="100">`

File: 1080i/script-skin_helper_service-ColorPicker.xml [hunk 5]
```diff
@@ -30,28 +30,28 @@
...
                     <control type="image">
                         <top>0</top>
                         <left>0</left>
-                        <right>10</right>
-                        <bottom>10</bottom>
+                        <right>8</right>
+                        <bottom>8</bottom>
```
Comment (jurialmunkey) [inline 1080i/script-skin_helper_service-ColorPicker.xml]: Same here. Can you revert the right/bottom pad to 10

**Review by jurialmunkey — CHANGES_REQUESTED** [review]
Looks good. Can you just make the requested changes on the lines that I've commented. Mostly just revert the padding back to its original values and change the itemlayout width/height to 100 so that you get more items fitting on the page.

Also, in `<control type="panel" id="3110">` change the bottom tag so that it is `<bottom>20</bottom>`

These changes will make sure that all items fit properly on the screen with the correct padding from sides/bottom of dialog.

Also, thanks for the PR. Really appreciate having improvements PR'd so I can just merge them! 😁

**Comment by drinfernoo** [comment]
I'll definitely make the changes! I wondered if maybe a toggle for "tile size" might be useful?

**Comment by drinfernoo** [comment]
I went ahead and fixed the padding for you 👍 I only changed it in the first place because Material Design uses baselines at 8-unit intervals... So I felt like it would be appropriate 😆

**Comment by jurialmunkey** [comment]
Cheers! Merged. 😁

**Comment by drinfernoo** [comment]
As an aside, do you think configurable cell layouts would be desirable or overkill?

**Comment by jurialmunkey** [comment]
Maybe if it was automatic and linked to which palette was selected, but it's probably overkill really.

**Pull request merged**